h2o-docs/src/product/data-science/if.rst

```diff
@@ -117,6 +117,42 @@ Defining an Isolation Forest Model
 
 - **contamination**: The contamination ratio is the proportion of anomalies in the input dataset. If undefined (``-1``), the predict function will not mark observations as anomalies and only anomaly score will be returned. Defaults to ``-1``.
 
+Anomaly Score
+~~~~~~~~~~~~~
+
+The output of Isolation Forest's algorithm depends on the ``contamination`` parameter.
+
+**With usage of the contamination parameter:**
+
+-  **predict**:
+
+    - ``1`` = Anomaly
+    - ``0`` = Normal point
+
+Point is marked as an anomaly if the score is great or equal to (1-``contamination``)% quantile of the score.
+
+.. math::
+    predict = score >= Q_{score}(1-contamination)
+
+-  **score**: Normalized mean_length
```

Comment:
Why do you use mean_length and then camelcase in maxPathLength and minPathLength? It looks strange.

```suggestion
-  **score**: Normalized meanLength
```

Reply:
I have changed it to snake case everywhere according to out conversation.